### ISSUE TITLE
Question: Accessability and Image alt tag

### ISSUE THREAD

**Issue by andy96453** [tecnickcom/TCPDF#468]
Hi,

Example 6 has alt="test alt attribute" but there are no alt tags in the pdf, are there any methods to add the alt tags for images to make the resulting pdf 508 compliant for accessibility?

Thanks

**Comment by nicolaasuni** [comment]
Thank you for reporting this. TCPDF 6 is now a legacy project and is officially **deprecated**, so no further fixes or features will be merged into this repository. As part of this transition, we are closing older issues. Please migrate to **[tc-lib-pdf](https://github.com/tecnickcom/tc-lib-pdf)** and, if the same problem still occurs there, open a new issue with updated reproduction steps. For more context, see the pinned announcement: [Thank You - and the Future of TCPDF](https://github.com/tecnickcom/TCPDF/issues/867).
